# Define the goal
geometry_msgs/PointStamped target_point
string direction
---
# Define the result
bool action_success
---
# Define a feedback message. 0 for not started, 1 for ongoing, 2 for failure.
uint32 feedback


